src/gpodder/gtkui/interface/common.py

```diff
@@ -154,7 +154,7 @@ def on_text_changed(editable):
         else:
             return None
 
-    def show_login_dialog(self, title, message, username=None, password=None,
+    def show_login_dialog(self, title, message, root_url=None, username=None, password=None,
```

Comment:
It looks like this can be hit when trying to restore from gPodder.net (can this only be done when there are no local podcasts?) and when encountering a podcast requiring authentication.

Do you know a simple/easy way to test each of these paths?